velox/connectors/hive/storage_adapters/hdfs/tests/InsertInfoHdfsTest.cpp

```diff
@@ -0,0 +1,235 @@
+/*
```

Comment:
file name has a typo, should be `InsertIntoHdfsTest.cpp`?

Reply:
Yes, this is a typo, I'll fix it.